dmsrc/time.dm

```diff
@@ -0,0 +1,3 @@
+#define rustg_time_microseconds(id) call(RUST_G, "time_microseconds")(id)
+#define rustg_time_milliseconds(id) call(RUST_G, "time_milliseconds")(id)
```

Comment:
You should maybe have these call `text2num` if they're going to return as strings